Source/AliveLib/PauseMenu.cpp

```diff
@@ -1011,6 +1016,85 @@ void PauseMenu::RestartPath()
 const char kArrowChar = 3;
 ALIVE_ARY(1, 0x55E398, char, 2, sArrowStr_55E398, { kArrowChar, 0 });
 
+#if CONSOLE_SAVE_MENU
+const int saveSlotCount = 8;
+
+std::string saveSlotTexts[saveSlotCount];
+std::vector<CustomPauseMenuItem> consoleSaveMenuItems;
+bool savePageToSave = true;
+void save_game_slot(int slot)
```

Comment:
probably the whole OG menu should be redone so that we don't need per platform menus

Reply:
agreed.